meta-intel-iot-middleware/recipes-devtools/iotkit-agent/iotkit-agent_1.7.0.bb

```diff
@@ -35,7 +35,7 @@ do_compile () {
     npm set cache ${WORKDIR}/npm_cache
 
     # clear local cache prior to each compile
-    npm cache clear
+    npm cache clear --force
```

Comment:
iotkit-agent is long deprecated and the cloud it was connecting towas discontinued as well - so feel free to drop this completely.

Reply:
Ok let's get rid of it